capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -27,63 +42,73 @@ class AdminClubPage extends Component {
     let data = {};
     const formElements = document.getElementById("update-club-form").elements;
     for (let i = 0; i < formElements.length; i++) {
-      if (formElements[i].name.length !== 0 && 
-          formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
+      if (formElements[i].name.length !== 0 &&
+        formElements[i].type !== "submit" && formElements[i].value.length !== 0) {
         data[formElements[i].name] = formElements[i].value;
       }
     }
     data.id = this.state.club.id;
     if (window.localStorage.getItem("userID") !== this.state.club.ownerID) {
       alert("Update failed. You do not own this club.");
       return;
-    } 
+    }
 
-    fetch("/api/clubs", {method: "put", body: JSON.stringify(data)})
-        .then(function() {
-          history.push(`/clubpage/${data.id}`);
-        })
-        .catch(function(e) {
-          console.log(e);
-          alert("Looks like we're having trouble connecting to our database, hang tight!");
-        });  
+    fetch("/api/clubs", { method: "put", body: JSON.stringify(data) })
+      .then(function () {
+        history.push(`/clubpage/${data.id}`);
+      })
+      .catch(function (e) {
```

Comment:
nit: Use arrow functions here for consistency with the rest of the codebase.

Reply:
Will do!